node/pkg/driver/node_test.go

```diff
@@ -895,23 +896,132 @@ func TestNodeUnpublishVolume(t *testing.T) {
 }
 
 func TestNodeGetVolumeStats(t *testing.T) {
```

Comment:
do you think we should add tests in `node_utils_test.go` as well?

Reply:
can't we just mock the executer calls (as was done in https://github.com/IBM/ibm-block-csi-driver/pull/450)?

Reply:
can I open a ticket for it and do it next sprint?

Reply:
sure, not urgent

Reply:
done CSI-3947